pipeline_plugins/resource/hook/celaction.py

```diff
@@ -9,8 +9,7 @@
 import _winreg
 
 if __name__ == "__main__":
```

Comment:
Nope, just noticed the path reference and felt it should all be cleaned up at the same time.